install/infra/modules/eks/kubernetes.tf

```diff
@@ -64,16 +98,16 @@ module "eks" {
   eks_managed_node_groups = {
     Services = {
       enable_bootstrap_user_data = true
-      instance_types = [var.service_machine_type]
-      name = "service-${var.cluster_name}"
-      subnet_ids   = module.vpc.public_subnets
-      min_size     = 1
-      max_size     = 10
-      desired_size = 1
-      block_device_mappings =  [{
+      instance_types             = [var.service_machine_type]
+      name                       = "service-${var.cluster_name}"
+      subnet_ids                 = module.vpc.public_subnets
+      min_size                   = 1
+      max_size                   = 10
+      desired_size               = 2
+      block_device_mappings = [{
         device_name = "/dev/sda1"
 
-        ebs =  [{
+        ebs = [{
```

Comment:
I think here we are recommending different settings for storage in the reg arch guide, namely:
```
    disableIMDSv1: false
    volumeSize: 300
    volumeType: gp3
    volumeIOPS: 6000
    volumeThroughput: 500
    ebsOptimized: true
    privateNetworking: true
    propagateASGTags: true
```
([source](https://www.gitpod.io/docs/self-hosted/latest/reference-architecture/single-cluster-ref-arch#kubernetes-cluster), see the gitpod-cluster.yaml file in the AWS tab of the cloud provider specific instructions)

--> I saw in the AWS cost that we are using GP2 storage and not GP3 storage in this current version of the terraform script. Not sure about the iops, throughput and the other stuff.